rpc/cc/arduino/cli/commands/v1/core.proto

```diff
@@ -115,7 +115,7 @@ message PlatformListReq {
 	bool all = 3;
 }
 
-message PlatformListResp {
+message PlatformListResponse {
 	// The installed platforms.
 	repeated Platform installed_platform = 1;
```

Comment:
`installed_platform` -> `installed_platforms` `repeated` field should be a plural: https://developers.google.com/protocol-buffers/docs/style#repeated_fields

Reply:
Good point! this is not detected by the linter, I've fixed it also in other places